presto-main/src/main/java/com/facebook/presto/sql/analyzer/AggregationAnalyzer.java

```diff
@@ -388,36 +414,42 @@ public Boolean visitWindowFrame(WindowFrame node, Void context)
         @Override
         protected Boolean visitIdentifier(Identifier node, Void context)
         {
-            return isField(QualifiedName.of(node.getName()));
+            return isField(node, QualifiedName.of(node.getName()));
```

Comment:
it probably also needs verification for columnReference (not lambda) if https://github.com/prestodb/presto/pull/7210 lands before